pkg/cli/jobflow/describe.go

```diff
@@ -0,0 +1,106 @@
+package jobflow
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/yaml"
+
+	"volcano.sh/apis/pkg/apis/flow/v1alpha1"
+	"volcano.sh/apis/pkg/client/clientset/versioned"
+	"volcano.sh/volcano/pkg/cli/util"
+)
+
+type describeFlags struct {
+	util.CommonFlags
+
+	// Name is name of jobflow
+	Name string
+	// Namespace is namespace of jobflow
+	Namespace string
+	// Format print format: yaml or json format
+	Format string
+}
+
+var describeJobFlowFlags = &describeFlags{}
+
+// InitDescribeFlags is used to init all flags.
+func InitDescribeFlags(cmd *cobra.Command) {
+	util.InitFlags(cmd, &describeJobFlowFlags.CommonFlags)
+	cmd.Flags().StringVarP(&describeJobFlowFlags.Name, "name", "N", "", "the name of jobflow")
+	cmd.Flags().StringVarP(&describeJobFlowFlags.Namespace, "namespace", "n", "default", "the namespace of jobflow")
+	cmd.Flags().StringVarP(&describeJobFlowFlags.Format, "format", "o", "yaml", "the format of output")
+}
+
+// DescribeJobFlow is used to get the particular jobflow details.
+func DescribeJobFlow(ctx context.Context) error {
+	config, err := util.BuildConfig(describeJobFlowFlags.Master, describeJobFlowFlags.Kubeconfig)
+	if err != nil {
+		return err
+	}
+	jobFlowClient := versioned.NewForConfigOrDie(config)
+
+	// Get jobflow list detail
+	if describeJobFlowFlags.Name == "" {
+		jobFlows, err := jobFlowClient.FlowV1alpha1().JobFlows(describeJobFlowFlags.Namespace).List(ctx, metav1.ListOptions{})
+		if err != nil {
+			return err
+		}
+		for _, jobFlow := range jobFlows.Items {
+			PrintJobFlowDetail(&jobFlow, describeJobFlowFlags.Format)
+			if len(jobFlows.Items) != 1 {
+				fmt.Println("---------------------------------")
+			}
+		}
+		// Get jobflow detail
+	} else {
+		jobFlow, err := jobFlowClient.FlowV1alpha1().JobFlows(describeJobFlowFlags.Namespace).Get(ctx, describeJobFlowFlags.Name, metav1.GetOptions{})
+		if err != nil {
+			return err
+		}
+		// Set APIVersion and Kind if not set
+		if jobFlow.APIVersion == "" || jobFlow.Kind == "" {
+			jobFlow.APIVersion = v1alpha1.SchemeGroupVersion.String()
+			jobFlow.Kind = "JobFlow"
+		}
+		PrintJobFlowDetail(jobFlow, describeJobFlowFlags.Format)
```

Comment:
Does `describe` just print jobflow's yaml?

Reply:
Yes. Same as `kubectl describe jobflow`

Reply:
Supports `-o yaml` or `-o json`, supports yaml format or json format. The default is yaml format